kpet/data.py

```diff
@@ -262,7 +262,8 @@ def __init__(self, data):
                     match=Class(PositivePattern),
                     dont_match=Class(NegativePattern),
                     url_suffix=String(),
-                    maintainers=NonEmptyList(String())
+                    maintainers=NonEmptyList(String()),
+                    max_runtime_minutes=Int(),
```

Comment:
We need to be able to assign different durations to different cases. Please either move this to the case, or add the same field to the case as well, and implement overriding suite duration with case duration in kpet-db.

Reply:
We need to follow this one up with a commit making it mandatory for the case to have a duration, after we add them in kpet-db. Otherwise we won't be able to implement scheduling. That would be easier if we allow only cases to have durations.

Reply:
Yes.

Also, I will followup.